anchore_engine/services/apiext/api/controllers/accounts.py

```diff
@@ -23,6 +23,7 @@
 from anchore_engine.db.db_account_users import UserAlreadyExistsError, UserNotFoundError
 from anchore_engine.utils import datetime_to_rfc3339
 from anchore_engine.common.helpers import make_response_error
+from anchore_engine.services.catalog.catalog_impl import load_policy_bundles
```

Comment:
Can we move this code into an "anchore_engine.subsys" location or part of "localconfig"? I'd like to avoid one service importing code from another service.